CHANGELOG.md

```diff
@@ -1,5 +1,38 @@
 Changes
 =======
+
+# 1.0.0 / 01-07-2020
+### Details
+https://github.com/jenkinsci/datadog-plugin/compare/datadog-0.7.1...datadog-1.0.0
+
+### Changes
+[BACKWARDS INCOMPATIBILITY NTOES]
```

Comment:
Also we should say that configurations got changed and should be revisited:
 - Groovy scripts need to be updated (descriptor path changed).  See [readme->configure-with-a-groovy-script](https://github.com/DataDog/jenkins-datadog-plugin/blob/master/README.md#configure-with-a-groovy-script)
 - More configs are available (some got removed, some added). See [readme->customization](https://github.com/DataDog/jenkins-datadog-plugin/blob/master/README.md#customization)
 - Whitelist and blacklist configs now support regex expressions